bookkeeper-server/src/main/java/org/apache/bookkeeper/server/Main.java

```diff
@@ -125,6 +135,8 @@ private static ServerConfiguration parseArgs(String[] args)
 
             ServerConfiguration conf = new ServerConfiguration();
 
+            loadConfEnv(conf);
+
             if (cmdLine.hasOption('c')) {
                 String confFile = cmdLine.getOptionValue("c");
                 loadConfFile(conf, confFile);
```

Comment:
Do you expect `loadConfFile` overrides `loadConfFile` if the same property is defined?

Reply:
Thank you for reminding me, my intention is that the environment variable has a higher priority than the configuration file.

I will modify it to call LoadConfEnv after loadConfFile

Reply:
Please add a command line option.

Otherwise with this change we are introducing unexpected behaviour and possibly a security issue.

Reply:
> Please add a command line option.
> 
> Otherwise with this change we are introducing unexpected behaviour and possibly a security issue.

Use the command-line option `env`, What do you think?